modules/phase_field/src/materials/ComputePolycrystalElasticityTensor.C

```diff
@@ -48,9 +48,6 @@ ComputePolycrystalElasticityTensor::computeQpElasticityTensor()
   const std::vector<std::pair<unsigned int, unsigned int> > & active_ops = _grain_tracker.getElementalValues(_current_elem->id());
```

Comment:
In terms of efficiency?

I'm just saying that we have two interfaces with same information but in different formats.
```
const std::vector<std::pair<unsigned int, unsigned int> > & getElementalValues(dof_id_type elem_id) const = 0;
const std::vector<unsigned int> & getOpToGrainsVector(dof_id_type elem_id) const = 0;
```

These are fairly large data structures and I'm currently constructing both of them inside of the grain tracker. I think we should drop one of them.